app/services/work_packages/set_attributes_service.rb

```diff
@@ -49,9 +49,7 @@ def set_attributes(attributes)
   end
 
   def mark_templated_subject
-    return true unless work_package.type&.replacement_patterns_defined?
-
-    if work_package.type.patterns.all_enabled[:subject]
+    if work_package.type&.replacement_pattern_defined_for?(:subject)
```

Comment:
This is the only place where there is a (small?) semantical change.

Previously  the method could have returned `true`, `nil` or `"Templated by ..."`. Now it will only return `nil` or `"Templated by ..."`. I am not sure if this is a meaningful difference that will break something, but the return value definitely escapes from the `set_attributes` method...